apps/web/src/components/chat/consent-prompt.tsx

```
@@ -4,9 +4,11 @@ import { useI18n } from '@/i18n/i18n';
 export function ConsentPrompt({ onAccept }: { onAccept?: () => void }) {
   const { t } = useI18n();
   return (
-    <div className="p-3 rounded border bg-yellow-50 text-yellow-800">
-      <p className="mb-2">{t('chat.consent_notice') || 'Para continuar, confirme o consentimento para uso de dados.'}</p>
-      <button className="px-3 py-1 rounded bg-yellow-600 text-white" onClick={onAccept}>
+    <div className='p-3 rounded border bg-yellow-50 text-yellow-800'>
+      <p className='mb-2'>
+        {t('chat.consent_notice') || 'Para continuar, confirme o consentimento para uso de dados.'}
+      </p>
+      <button className='px-3 py-1 rounded bg-yellow-600 text-white' onClick={onAccept}>
         {t('chat.consent_accept') || 'Aceitar'}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)